api/RTCIceCandidate.json

```diff
@@ -770,10 +770,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceCandidate/usernameFragment",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
Set to false in https://github.com/mdn/browser-compat-data/pull/3689. Support was added in https://chromium.googlesource.com/chromium/src/+/925f0a11ff18ee7403d044b6d3a4efbe639bf4cd which will be in Chrome 74. Using "74" would be OK I think, but for Chromium-based browsers where you don't know what the next version including M74 is, I think false or null is still best.

Reply:
I agree that false/null would be the best option, since we have no way of predicting the future.  I'll set it to false!